reference/constraints/Hostname.rst

```diff
@@ -0,0 +1,135 @@
+Hostname
+========
+
+This constraint ensures that the given value is a valid and non-reserved
+host name (internally it uses the ``FILTER_VALIDATE_DOMAIN`` option of the
```

Comment:
could you link the constant?

Reply:
I'm afraid there's no link to the constant in the PHP manual. There's only this page: https://www.php.net/manual/en/filter.filters.validate.php